### PR TITLE
update(capacity) - new source for FI solar capacity 

### PR DESCRIPTION
**Pull request by mathilde-daugy**
## Issue
see issue #6490 

## Description

- Create capacity for FI that merges ENTSOE capacity and FINGRID capacity (solar and wind)
- Updated config

### Preview

image

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: electricitymap/contrib/capacity_parsers/FI.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+from datetime import datetime, timedelta
+from typing import Any
+
+from requests import Response, Session
+
+from electricitymap.contrib.capacity_parsers.ENTSOE import (
+    fetch_production_capacity as fetch_entsoe_production_capacity,
+)
```
Comment (VIKTORVAV99) [inline electricitymap/contrib/capacity_parsers/FI.py:8]: I'm a little concerned about importing other parsers without having some form of testing in place to ensure we don't break other parsers. 

For example I have https://github.com/electricitymaps/electricitymaps-contrib/pull/6496 open which could change how things are working in here.

**Comment by mathilde-daugy** [comment]
will look at testing on monday

**Comment by mathilde-daugy** [comment]
@VIKTORVAV99 I see that you resolved your own comment on this PR. Is there anything you need me to do in particular or can we merge this?

**Comment by VIKTORVAV99** [comment]
Ah no, I'll review it right away, I had just kind of forgotten about this 😬

File: electricitymap/contrib/capacity_parsers/FI.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+from electricitymap.contrib.config import ZoneKey
+
+API_KEY = "a411d83c96bb4abb8812dba80bdaed64"
```
Comment (VIKTORVAV99) [inline electricitymap/contrib/capacity_parsers/FI.py:11]: We really should not store the API key here. It should be a ENV variable.

(PS: you should revoke this API key if possible as it's already in the git history).

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]

File: electricitymap/contrib/capacity_parsers/FI.py (hunk 2 above)
Comment (mathilde-daugy) [reply electricitymap/contrib/capacity_parsers/FI.py]: done!

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM!

**Pull request merged**